pkg/kubelet/server/stats/summary_sys_containers.go

```diff
@@ -80,7 +83,11 @@ func (sp *summaryProviderImpl) GetSystemContainersCPUAndMemoryStats(nodeConfig c
 		}
 		s, err := sp.provider.GetCgroupCPUAndMemoryStats(cont.name, cont.forceStatsUpdate)
 		if err != nil {
-			klog.ErrorS(err, "Failed to get system container stats", "containerName", cont.name)
+			if errors.Is(errors.Unwrap(err), cadvisormemory.ErrDataNotFound) {
```

Comment:
you should be able to use `errors.Is` without `errors.Unwrap` https://pkg.go.dev/errors#Is right?

Reply:
Right. Just fixed. Thanks.